cmd/kubeadm/app/util/config/testdata/defaulting/master/defaulted.yaml

```diff
@@ -115,7 +115,6 @@ evictionHard:
   imagefs.available: 15%
   memory.available: 100Mi
   nodefs.available: 10%
-  nodefs.inodesFree: 5%
```

Comment:
Test failure here:
```
--- FAIL: TestConfigFileAndDefaultsToInternalConfig (0.94s)
    --- FAIL: TestConfigFileAndDefaultsToInternalConfig/incompleteYAMLToDefaultedv1beta1 (0.00s)
        masterconfig_test.go:123: the expected and actual output differs.
                        in: testdata/defaulting/master/incomplete.yaml
                        out: testdata/defaulting/master/defaulted.yaml
                        groupversion: kubeadm.k8s.io/v1beta1
                        diff: 
                --- expected
                +++ actual
                @@ -115,6 +115,7 @@
                   imagefs.available: 15%!
                (MISSING)   memory.available: 100Mi
                   nodefs.available: 10%!
                (MISSING)+  nodefs.inodesFree: 5%!
                (MISSING) evictionPressureTransitionPeriod: 5m0s
                 failSwapOn: true
                 fileCheckFrequency: 20s
    
FAIL
FAIL    k8s.io/kubernetes/cmd/kubeadm/app/util/config   1.007s
```

There is probably some change in the KubeletConfig here. So whenever a change is made there we need to update this. We may want to just check if we have the right `kind` here. Yet, this is a job for another PR.